### PR TITLE
Update release 0.5.0 rc3

### PR DESCRIPTION
**Pull request by g-krisztian**
## Title: 
Update version for 0.5.0-rc3 release

### Description
I'd like to make a release with the latest changes

### Tester info
A brief description of any steps needed to demonstrate the new functionality or show the bug is fixed

### Completion Checklist

- [x] Add description of the changes made here to the changelog file

### CODE REVIEW

**Comment by soulflyer** [comment]
Should we have the new API docs generated for this PR or as a separate PR?

**Comment by g-krisztian** [comment]
> Should we have the new API docs generated for this PR or as a separate PR?

I'm working on the new documentation:
https://github.com/Flexiana/framework/tree/update_documentation

**Comment by soulflyer** [comment]
So that will be a separate PR?

**Comment by g-krisztian** [comment]
> So that will be a separate PR?

Yes, it will be another PR.

**Review by agigao — APPROVED** [review]

File: doc/tutorials.md [hunk 1]
```diff
@@ -67,70 +67,65 @@ The system configuration and start-up with the chainable set-up:
...
 
-In migratus library there is an Achilles point:
+Database migration is based on the following principles:
 
-It has no option to define separate migrations by profiles. Xiana
-decorates [Migratus](https://github.com/yogthos/migratus), to handle this weakness.
+1. The migration process is based on a stack of immutable changes. If at some point you want to change the schema or the
+   content of the database you don't change the previous scripts but add new scripts at the top of the stack.
+2. There should be a single standard resources/migrations migration directory
+3. If a specific platform (dev, stage, test, etc) needs additional scripts, specific directories should be created and
```
Comment (jehaby) [inline doc/tutorials.md:75]:
```suggestion
3. If a specific environment (dev, stage, test, etc) needs additional scripts, specific directories should be created and
```

**Review by jehaby — APPROVED** [review]
lgtm except a small suggestion

**Pull request merged**